library/Rules/Date.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/*
+ * This file is part of Respect/Validation.
+ *
+ * (c) Alexandre Gomes Gaigalas <alexandre@gaigalas.net>
+ *
+ * For the full copyright and license information, please view the "LICENSE.md"
+ * file that was distributed with this source code.
+ */
+
+namespace Respect\Validation\Rules;
+
+use DateTimeInterface;
+
+class Date extends AbstractRule
+{
+    public $format = null;
+
+    public function __construct($format = null)
+    {
+        $this->format = $format;
```

Comment:
This property should not be even set if `$format` doesn't match `/[djSFmMnYy\W]+$/`